src/NCronJob/Registry/JobRegistry.cs

```diff
@@ -155,6 +156,29 @@ Job registration conflict detected. A job has already been registered with the n
             """);
     }
 
+    private void AssertOnlyOneUnnamedUnscheduledParameterizedTypedJob(JobDefinition jobDefinition)
+    {
+        if (jobDefinition.CustomName is not null
```

Comment:
Can we put this onto the `JobDefintion type`? That would be then also super trivial to unit test

Reply:
I will. But this will be one hell of a property: `IsUnnamedOrUnscheduledOrParameterlessTypedJob` :wink:

Reply:
@linkdotnet Fixed. And rebased.